### DISCUSSION TITLE
Quantas vezes é possivel abrir um MEI?

### DISCUSSION THREAD

**Discussion by elMor3no** [DyegoCosta/trabalhando-remoto#165] [Q&A]
Hola galera! 

Tenho uma grande duvida, falando com um colega sobre a possibilidade de me cadastrar como MEI ele me falou que caso eu cancele meu cadastro MEI eu não poderia voltar a cadastrar como MEI nunca mais. Pesquisei mas se bem não fala nada sobre não poder abrir um novo MEI tampouco fala nada sobre poder abrir, ai fiquei na duvida.   

Alguém já cancelou seu cadastro e voltou a abrir ele novamente?

**Comment by igorlvicente** [comment]
Já tive 2 MEIs, mas te aconselho a perguntar pra quem entende, mesmo que isso custo alguns reais, porque se for algo que pode dar problema pra você, o barato sai caro.
